package.json

```diff
@@ -179,7 +179,7 @@
     "fast-json-stringify": "^2.5.2",
     "fastify-error": "^0.3.0",
     "fastify-warning": "^0.2.0",
-    "find-my-way": "^4.1.0",
+    "find-my-way": "^5.1.0",
```

Comment:
We can't land this change to `main`.

We should ask @delvedor to backport the fix to 4.x or targeting the `next` branch

Reply:
> We should ask @delvedor to backport the fix to 4.x or targeting the next branch

@Eomm @luisorbaiceta would you like to open a PR applying the fixes in https://github.com/delvedor/find-my-way/pull/214 to https://github.com/delvedor/find-my-way/tree/4.x? You can likely cherry-pick the commit

Reply:
Sure! @mcollina